### PR TITLE
add basic CI and dependabot

### LINKED ISSUE #13: Setup CI
Preferably github actions

### PR DESCRIPTION
**Pull request by ordian**
Fixes #13

### CODE REVIEW

**Comment by ordian** [comment]
@drahnr if you don't have any objections, I'm merging this :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43 # v2.7.0
+
+      - run: rustup target add wasm32-unknown-unknown
+
```
Comment (ordian) [inline .github/workflows/ci.yml]:
```suggestion
```

**Comment by drahnr** [comment]
Before you merge, could you do a 1.0.2 release so there's a clear pre version of this PR?

Other than that, thank you 🙏

**Comment by ordian** [comment]
sure, do you want to update your #11 branch? i can publish from there or if you prefer, you can also do that :)

**Pull request merged**